platform-api/che-core-api-auth/src/main/java/org/eclipse/che/security/oauth/OAuthAuthenticationService.java

```diff
@@ -236,6 +239,20 @@ public Response invalidate() throws IOException {
                                                                            null)).type(MediaType.TEXT_PLAIN).build();
     }
 
+    @GET
+    @Produces({MediaType.APPLICATION_JSON})
+    @RolesAllowed({"user", "temp_user"})
+    @Path("token")
```

Comment:
I think <code>@Path</code> should be declared after Request type _(GET)_ in this case, as for me it is more logical sequence, see examples [here](https://github.com/codenvy/che-core/blob/4.0/platform-api/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java) and  [here](https://github.com/codenvy/che-core/blob/4.0/platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/command/CommandService.java)